gulpfile.js

```diff
@@ -0,0 +1,101 @@
+var _ = require('lodash');
+var gulp = require('gulp');
+var gutil = require('gulp-util');
+var ts = require('gulp-typescript');
+var tslint = require('gulp-tslint');
+var realTslint = require('tslint');
+var spawnSync = require('child_process').spawnSync;
+
+function build(src, dest) {
+  var tsProject = ts.createProject('configs/tsconfig-base.json', { declaration: true });
+  return gulp.src(src)
+    .pipe(tsProject())
+    .pipe(gulp.dest(dest));
+}
+
+gulp.task('build:capnp', function () {
+  return build('./packages/capnp-ts/src/**/*.ts', 'packages/capnp-ts/lib');
+});
+
+gulp.task('build:capnpc', function () {
+  // Stub, schema are not compiled yet
+  return build('./packages/capnpc-ts/src/**/*.ts', 'packages/capnpc-ts/lib');
+});
+
+gulp.task('build', ['build:capnp'/*, 'build:capnpc'*/]);
```

Comment:
I just tried uncommenting it and it seems to be building just fine (everything in `lib/` looks ok).

I do get an annoying warning from the typescript compiler though for each package:

```
error TS18003: No inputs were found in config file 'tsconfig-base.json'. Specified 'include' paths were '["**/*"]' and 'exclude' paths were '["node_modules","bower_components","jspm_packages"]'.
```

Not sure if that's worth putting in effort to fix right now; it seems harmless.